recipes/microprofile/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.10)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+
+target_link_libraries(${PROJECT_NAME} PRIVATE CONAN_PKG::microprofile)
```

Using a `CONAN_PKG::` target serves no purpose in the test package.
So please revert to the "easy" default syntax.
```suggestion
conan_basic_setup()

add_executable(${PROJECT_NAME} test_package.cpp)
target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
```